charts/aws-ebs-csi-driver/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.9.0"
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
-version: 0.9.8
+version: 0.9.9
```

Comment:
@ayberk PTAL

Reply:
I just merged an earlier PR, could you update it to 0.9.10?

Reply:
ok